gradle-jdks-setup/README.md

```diff
@@ -211,6 +211,28 @@ The plugin registers the following tasks:
 - `checkGradleJdkConfigs` - checks that all the `gradle/` configurations are up-to-date. E.g. if the `jdks-latest` plugin is updated, we need to make sure the `gradle/jdks` files reflect the jdk versions.
 - `setupJdks` - task that triggers `wrapperJdkPatcher` and `generateGradleJdkConfigs` and runs the patched `./gradlew` script.
 
+
+## Using a new java version
+
+We are only generating the jdk configuration files for the jdk versions that are used see: [jdkMajorVersionsToUse](https://github.com/palantir/gradle-jdks/blob/26d54605b7827b232024ec7e0c34a87983804492/gradle-jdks/src/main/java/com/palantir/gradle/jdks/JdksExtension.java#L44)
+Using a new java version is a 2-step process: 
+1. Firstly, we need to generate the gradle jdk configuration files for the new java version that we are going to use
+2. Secondly, once the new files are generated, the javaVersion can be configured in the gradle build configuration.
+
+### Palantir specific, if using `baseline-java-version` through excavators
+1. Run `./gradlew generateGradleJdkConfigs --includeAllJdks`. The command will generate the Gradle jdk configuration files for all jdks.
+2. Update the javaVersion(s)/jdks extension values with the desired java major version.
+3. Run `./gradlew setupJdks` to only keep the Gradle jdk configuration files for the used Gradle JDKs.
+
+### Manual bumps of the java versions 
+1. Run `./gradlew generateGradleJdkConfigs --includeVersion=<newVersion> [--includeVersion=<otherNewVersion>]`. The command will generate the Gradle jdk configuration files for the major versions passed as parameters.
+2. Update the javaVersion(s)/jdks extension values with the desired java major version.
```

Comment:
We still need to run `./gradlew generateGradleJdksConfigs` even with this approach as the user may stop using the old version, and we need to remove it

Reply:
Addressed, I thought I had the logic in the `check` task to test if there are any extra configuration files in gradle/jdks. I added now that logic in: https://github.com/palantir/gradle-jdks/pull/479/commits/1710e98961e625581cc0d668686f61f972f1d899#diff-53f3309ab573fe65d62f8b5bf40e9bb94478d82b4d6434cb6bcf9b25516813a4R134